disnake/integrations.py

```diff
@@ -409,13 +409,18 @@ class BotIntegration(Integration):
         The integration account information.
     application: :class:`IntegrationApplication`
         The application tied to this integration.
+    scopes: List[:class:`str`]
```

Comment:
Would it make sense to turn this into a set?

Reply:
Sure, sounds good.
https://github.com/DisnakeDev/disnake/pull/729/commits/1c3be76b2d6c3a20692301dc6d1c36321f0e7127

Reply:
That was a question, I wasn't sure it was a good idea! 😅

Reply:
Additionally, we use Lists elsewhere for scopes already so we should keep it a lsit.